src/Components/pages/Category.js

```diff
@@ -1,15 +1,19 @@
 import React from 'react';
+import { useDispatch } from 'react-redux/es/exports';
+import { addNewCtegoryFunction } from '../../redux/categories/categories';
 
 const propTypes = {};
 
 const defaultProps = {};
 
 const Category = () => {
-  const handleTestRedux = () => {
+  const dispatch = useDispatch();
+  const handleCheckStatus = () => {
+    dispatch(addNewCtegoryFunction());
   };
   return (
     <div>
-      <button type="button" onClick={handleTestRedux}>Check status</button>
+      <button type="button" onClick={handleCheckStatus}>Check status</button>
     </div>
   );
 };
```

Comment:
- [ ] [Optional] It would be nice to access the `redux` store and retrieve the `'under construction'` data which would then be displayed using a JSX. 👍